sports_club/data/payment_provider_demo.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <function model="payment.provider" name="write">
```

Comment:
This shouldn't be a function, but an update on the record.
Also, you should add the opposite behaviour to activate it back in a file in demo.

Reply:
Thanks for the quick suggestion @vava-odoo
Changes has been done accordingly